### PR TITLE
Docs update for new features

### PR DESCRIPTION
**Pull request by tomhuang12**
- Added runner pod spec docs
- Added force-unlock docs

### CODE REVIEW

File: docs/use_cases.md [hunk 1]
```diff
@@ -537,3 +542,29 @@ spec:
   writeOutputsToSecret:
     name: helloworld-outputs
 ```
+
+## Force unlock Terraform state
```
Comment (tomhuang12) [inline docs/use_cases.md:546]: @Nalum I added some docs for the force-unlock feature. Could you take a look to see if this aligns with what you implemented? Thanks!

**Comment by tomhuang12** [comment]
@chanwit any other notable use cases you would like me to add a doc for?

**Comment by chanwit** [comment]
@tomhuang12, thank you so much.
Would it be possible for you to help write the doc for `spec.storeReadablePlan`, please?

**Review by chanwit — APPROVED** [review]

**Comment by tomhuang12** [comment]
@chanwit sorry I was on vacation last week. I will work on it on a separate PR.

**Comment by chanwit** [comment]
No problem, @tomhuang12 :+1:

**Pull request merged**